content/en/docs/tasks/security/authentication/mtls-migration/index.md

```diff
@@ -145,11 +150,75 @@ If you can't migrate all your services to Istio (injecting Envoy sidecar), you h
 However, when configured with `PERMISSIVE` mode, no authentication or authorization checks will be performed for plaintext traffic by default.
 We recommend you use [Istio Authorization](/docs/tasks/security/authorization/authz-http/) to configure different paths with different authorization policies.
 
+## Globally enabling mutual TLS for the cluster
```

Comment:
If I understand correctly, this step is to migrate all workloads to use mTLS at once, and the above 2 steps is to migrate a specific workload to use mTLS. Should we make it clear that these are 2 separate migration path and their relationship is parallel?

Reply:
Yes, we need to make that clear. I also found that we need to further improve this task. The task gives instructions based on the task. We need to tell customers with existing clusters how to inspect their system and migrate.